reflex-dom-core/src/Reflex/Dom/Builder/Static.hs

```diff
@@ -257,6 +259,18 @@ instance SupportsStaticDomBuilder t m => NotReady t (StaticDomBuilderT t m) wher
   notReadyUntil _ = pure ()
   notReady = pure ()
 
+prependCallStackComment :: DomBuilder t m => CallStack -> m a -> m a
+prependCallStackComment cs w = do
+  let
+    removeSelf :: [([Char], SrcLoc)] -> [([Char], SrcLoc)]
+    removeSelf = filter $ not . ("reflex-dom-core" `isPrefixOf`) . srcLocPackage . snd
```

Comment:
`reflex-dom-core` shouldn't be hardcoded - but how?
Using [`packageName`](https://hackage.haskell.org/package/base-4.12.0.0/docs/GHC-Generics.html#v:packageName)? 
Adding a `getPackageName :: CallStack` function that wraps a `let`/`where` binding with `HasCallStack =>` to grab it from there?
Template Haskell?